### PR TITLE
Freetype2 update to 2.13.0

### PR DESCRIPTION
**Pull request by jellefoks**
Update FreeType from 2_10_2 to 2_13_0
    
b/172481249

### CODE REVIEW

File: third_party/freetype2/autogen.sh [hunk 1]
```diff
@@ -162,4 +172,29 @@ cd ../..
...
+}
+
+if test -e ".git"; then
+  DLG_INC_DIR=subprojects/dlg/include/dlg
+  DLG_SRC_DIR=subprojects/dlg/src/dlg
+
+  if test -d "$DLG_INC_DIR"; then
+    :
+  else
+    echo "Checking out submodule in \`subprojects/dlg':"
```
Comment (isarkis) [inline third_party/freetype2/autogen.sh:192]: I found all workspace on win32 runners have this code and I think it's causing git checkout problem.

File: third_party/freetype2/.gitmodules
Comment (isarkis) [inline third_party/freetype2/.gitmodules:1]: I think this file is causing issues with git checkout.

Comment (kaidokert) [reply third_party/freetype2/.gitmodules:1]: Lets remove this file entirely for now - i don't think it's needed in our repo. If dlg is a must have dependency, we'll need to separately subtree it in.

Comment (jellefoks) [reply third_party/freetype2/.gitmodules:1]: The file is removed as part of the PR "Freetype2 Starboardization and customization for Cobalt." https://github.com/youtube/cobalt/pull/361

-> https://github.com/youtube/cobalt/pull/361/commits/c6a1c8d30de46bc097eb293aedb932d912d34492

**Comment by briantting** [comment]
This should only have 2 commits in total.

**Comment by jellefoks** [comment]
Superseded by https://github.com/youtube/cobalt/pull/381

**Pull request closed without merging**